app/views/home/index.html.erb

```diff
@@ -6,27 +6,18 @@
     </div>
   </div>
   <!-- "Welcome" section -->
-  <section class="homepage-welcome-banner dm-gradient-banner-318" role="region" aria-label="Homepage welcome banner">
-    <div class="grid-container height-full">
-      <div class="grid-row height-full flex-align-center">
+  <section class="homepage-welcome-banner usa-section padding-top-3 padding-bottom-7" role="region" aria-label="Homepage welcome banner">
+    <div class="grid-container">
+      <div class="grid-row">
         <div class="grid-col-12">
-          <h1 class="usa-prose-h1 text-white margin-top-4 margin-bottom-4 desktop:margin-top-0">
+          <h1 class="usa-prose-h1 text-white">
             Discover VA innovations to adopt at your facility
           </h1>
-          <div class="grid-row position-relative">
-            <div class="homepage-welcome-banner-body-text-container grid-col-12 desktop:grid-col-7 padding-right-105">
-              <p class="usa-prose-body text-white margin-bottom-2">
+          <div class="grid-row">
```

Comment:
remove this extra `grid-row` and make `homepage-welcome-banner` a sibling of the header